src/commands.rs

```diff
@@ -127,6 +131,46 @@ impl ScreenCommand {
 
         Ok(Screens::new(serde_json::from_str(&response.text()?)?))
     }
+
+    pub fn add(
+        &self,
+        pin: &str,
+        maybe_name: Option<String>,
+    ) -> anyhow::Result<Screens, CommandError> {
+        let url = self.authentication.config.url.clone() + "/v3/screens/";
+        let mut payload = HashMap::new();
+        payload.insert("pin".to_string(), pin.to_string());
```

Comment:
If pin is anyway converted from &str to String - may be function should accept String instead.

Reply:
I think it's a best practice to use &str but I am not 100 percent sure